### PR TITLE
Feature/mutect filter (Adding command for mutect filter)

### PR DESCRIPTION
**Pull request by rnaidu**
Added code, documentation, and command(s) to post processing to call paired-sample filtering of input VCF files derived from MuTect software v.1.1.5

Input command:
"pv mutect paired-sample filter --help"

Input file(s) needed:

- input MuTect VCF file
- input MuTect txt standard file

### CODE REVIEW

**Comment by rnaidu** [comment]
> My 2 cents
> 
> * [x]  we should use mutect1
> * [x]  command should follow same structure as vardict `pv mutect1 filter case-control`

good idea, changes have been made/committed!

File: postprocessing_variant_calls/mutect/mutect_process.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# imports 
+from __future__ import division
```
Comment (buehlere) [inline postprocessing_variant_calls/mutect/mutect_process.py:3]: we should remove this and make sure division still works.

File: postprocessing_variant_calls/mutect/mutect_class.py [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+def _tumor_variant_calculation(trd,tad):
+    ##############################
+    # Tumor Variant Calculations #
+    ##############################
+
+    tdp = trd + tad
+
+    if tdp != 0:
+        tvf = int(tad) / float(tdp)
```
Comment (buehlere) [inline postprocessing_variant_calls/mutect/mutect_class.py:209]: We should also consider writing division error cases. For example, division by zero.

File: postprocessing_variant_calls/mutect/mutect_class.py [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+    return tdp,tvf
+
+def _normal_variant_calculation(nrd,nad):
+    ###############################
+    # Normal Variant Calculations #
+    ###############################
+
+    ndp = nrd + nad
+    if ndp != 0:
+        nvf = int(nad) / float(ndp)
```
Comment (buehlere) [inline postprocessing_variant_calls/mutect/mutect_class.py:222]: We should also consider writing division error cases. For example, division by zero.

**Review by buehlere — CHANGES_REQUESTED** [review]
Great work! A lot of this review is related to updating the filter script from what was there previously and improving overall code quality in the package.

File: test.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+testing
```
Comment (buehlere) [inline test.txt:1]: we'll want to remove this. Could you also add a pytest like we do here: https://github.com/msk-access/postprocessing_variant_calls/blob/main/tests/test_maf.py? Just testing if the command `--help` works is fine. No need to use real data.

**Review by buehlere — APPROVED** [review]

**Pull request merged**